### DISCUSSION TITLE
GSoD 2023 discussion

### DISCUSSION THREAD

**Discussion by ggold7046** [micropython/micropython#11210] [General]
Hello,

I'm interested in this year's season of Docs program. Could anyone guide me how to participate?

**Comment by mattytrentini** [comment]
Read the [MicroPython Season of Docs 2023 proposal](https://docs.google.com/document/d/1JtwBneelTCVpFIw4ER0POs_-jksD96I00qqanieELMQ/edit?usp=sharing) that outlines what we're trying to achieve and, if that sounds interesting to you as a Technical Writer, apply by sending me (matt.trentini@gmail.com) an email with your Statement of Interest. 

More information is available in the GSoD [Technical writer guide](https://developers.google.com/season-of-docs/docs/tech-writer-guide).

**Comment by ggold7046** [comment]
Hello Matt,

Thanks for the reply. I wanted to know if  you have any project specific
application template for the technical writer position for GSoD 2023?

Thanks,
Aritra Goswami

**Reply by mattytrentini** [reply]
No, just use the template as supplied by the Season of Docs team (it's in the Technical writer guide, above).

**Comment by Bhajneet-Singh-Bedi** [comment]
Hey @mattytrentini is there any deadline to submit the template?

**Comment by ggold7046** [comment]
Hi Matt,

What is the last date for application to Micropython GSoD ?
